src/utils/format.ts

```diff
@@ -6,18 +6,25 @@ import { TEZ, TEZ_DECIMALS } from "./tezos";
 export const truncate = (name: string, len: number) =>
   name.length > len ? name.slice(0, len - 3) + "..." : name;
 
-export const tezToMutez = (tez: string): BigNumber => format("tz", "mutez", tez) as BigNumber;
+export const tezToMutez = (tez: string): string | number | BigNumber => format("tz", "mutez", tez);
 
-export const mutezToTez = (mutez: BigNumber | string) => format("mutez", "tz", mutez) as string;
+export const mutezToTez = (mutez: BigNumber | string | number) =>
+  format("mutez", "tz", mutez) as string;
 
-export const prettyTezAmount = (mutez: BigNumber | string): string => {
-  const tezAmount = BigNumber(mutezToTez(mutez)).toNumber();
+const formatTezAmount = (mutez: BigNumber | string | number): string => {
+  const tezAmount = BigNumber(mutezToTez(mutez || "0")).toNumber();
```

Comment:
why `|| '0'`?

Reply:
to avoid this case
<img width="544" alt="image" src="https://github.com/trilitech/umami-v2/assets/169451151/cd9f7a50-5ea9-4ec2-be37-00d369e5c323">